### DISCUSSION TITLE
Altimeter

### DISCUSSION THREAD

**Discussion by geologistluna** [CaptainBobSim/The-Cessna-172-Project-V3#226] [Q&A] [answered]
I have built several of the six pack so far, and am working on printing the Altimeter now, however, I am having a difficult time figuring out how some of this will come together, do you have a 3D breakdown of how it all goes together?  Maybe an exploded view of the altimeter? Thank you, I appreciate it.

**Comment by CaptainBobSim** [comment, marked as answer]
Hi geologistluna!
I'm glad you are working through the project!
It is still in prototype phase - It might work but it'll likely be terrible.
I can't provide support for it until the design is done. 
Files that have this warning are in progress:
[WARNING - THIS FOLDER IS IN PROTOTYPE STAGE.md](https://github.com/CaptainBobSim/The-Cessna-172-Project-V3/files/10868163/WARNING.-.THIS.FOLDER.IS.IN.PROTOTYPE.STAGE.md)

I've renamed the folders to be less confusing :)

Let me know if you have any more struggles!

**Reply by geologistluna** [reply]
Please let me know what I can do to assist.  I see the design more and more as I work with it, and I can see what you are trying to do, and I do see that you want to put a sensor onto it (neodymium magnet). I don't know what I can do, or how you may need assistance from me.  I am a pilot and understand the workings of the pneumatic instruments in the 172/182's. I used to have full access to Fusion 360, but the school I work at apparently is not on Fusion 360's list any longer (I will keep looking).  

If you are not using Fusion 360, let me know what it is, as I need a good design platform now that my F360 subscription is out.  

Thanks so very much.

**Comment by CaptainBobSim** [comment]
Absolutely! What are you interested in working in? The Altimeter?
My design now is going to use a Potentiometer to sense the card position, as a hall effect sensor would be difficult to implement. 
Can you export .STEP files with a hobbyist license?

**Comment by geologistluna** [comment]
I do not know if F360 will allow me a Hobbyist license since had been using
the education license for so long.  I will attempt to keep getting the
education license.  I sure don't want to pay the $70/mo unless I have an
income that covers the program.

I am interested in any and all Aviation related work.  I would not mind
coming up with an altimeter design.  It's a worthwhile challenge. I've seen
some amazing designs that used the original gear ratios, and I can only
imagine how difficult it is to try to mimic the barometer/aneroid
interaction with the Kollsman plate. A potentiometer could be appropriate,
however, a good, high sensitivity, low background noise pot can be
expensive.  I've found a couple that I've used for guitar amps, but they
already have some electronics attached.... An interesting problem to
address.

Antonio Luna Ph.C.
University of South Florida/Tampa

This email is not used for business, if you need to discuss business or
government related business, please contact me at my business email.

**Reply by CaptainBobSim** [reply]
Gotcha - If you really need to you can create a new account with a different email to keep them separate. I can send you my files.
I hear you about the cost - it's expensive!

I've seen some cool designs with gear ratios, but I opted for 3 steppers so that it could go from 0 feet to 5,000 feet more quickly. (1000s foot needle goes from 0 to 5 instead of making 5 full rotations)
I'm using a 10 turn potentiometer for the plate, that allows a good amount of accuracy.

**Comment by geologistluna** [comment]
I have a temp license with F360 and will keep working to get the full educator's license.  

One of the stepper motor mounts (10,000') is interfering with the insertion of another stepper motor (1,000'), so there will need to be some trimming.  If you would like me to work on that, send it my way.

**Comment by geologistluna** [comment]
The 10-turn precision potentiometer is a very good choice, and likely, you will not need the range it offers. I am avoiding all eBay sales as they are remarkably unreliable, but instead have an account with Mouser.com, and they have a very large range of potentiometers.  I'm not fully familiar with the requirements of the Arduino (yet) so I am not certain what kind of POT to consider designing into the altimeter.

**Comment by CaptainBobSim** [comment]
Could you send a picture of the issue?
Doing my assembly I couldn't find an issue with anything.
I'm using a Bournes 10K Potentiometer :)

**Reply by geologistluna** [reply]
![20230310_133829](https://user-images.githubusercontent.com/76762611/224406002-c5716faa-3e86-4152-b19c-ac9bfd5e8168.jpg)
![20230310_133958](https://user-images.githubusercontent.com/76762611/224406013-70e01773-8c10-43f9-80ec-e98ee3b911f8.jpg)
![20230310_133919](https://user-images.githubusercontent.com/76762611/224406024-2bb92b36-4c29-4fb9-acf4-df4ba8a55595.jpg)

In the last image (20230310_133919.jpg) you can see that the stepper motor below the deck does not have the clearance because the screw column for the 10,000' motor is protruding into the opening.

**Comment by CaptainBobSim** [comment]
Gotcha - I'll fix this :)
![image](https://user-images.githubusercontent.com/76851401/224414339-513f866b-2184-4b07-9482-a0300642d479.png)

**Comment by CaptainBobSim** [comment]
Fixed!

**Reply by geologistluna** [reply]
Can I get a copy of the .stl to print it and test it?

**Reply by CaptainBobSim** [reply]
Could you remind me in a week when I get back from vacation?

**Reply by CaptainBobSim** [reply]
I managed to sign in :) @geologistluna I updated the github repo with the new file

I'm finding out there's far too many issues in my current design so I'm going to have to do a major overhaul
They are all listed here:
https://github.com/CaptainBobSim/The-Cessna-172-Project-V3/issues/176
You can add a new comment to the issue if you find any more problems 🙃

Thank you for bringing your ideas to the table!!

**Comment by CaptainBobSim** [comment]
![image](https://user-images.githubusercontent.com/76851401/224725811-24bef7e0-acbb-4c0f-b605-2f2cd3d46ad6.png)
Here's a preview of the overhaul. I wanted to make the gears all the same size for ease of assembly. 
This also allows you to more easily screw it into the panel as the screwholes aren't blocked by gears.
